browser/src/control/Control.JSDialog.js

```diff
@@ -372,10 +372,16 @@ L.Control.JSDialog = L.Control.extend({
 	addFocusHandler: function(instance) {
 		if (!instance.canHaveFocus)
 			return;
+		
+		const elementToFocus = document.getElementById(instance.init_focus_id);
+
+		if (instance.init_focus_id === 'input-modal-input' && elementToFocus) {
```

Comment:
I would check for "input" html element type to target all the text fields, not an explicit id. More flexible